src/spec/extensionmetadocgenerator.py

```diff
@@ -133,7 +133,12 @@ def beginFeature(self, interface, emit):
                 write('  - Requires <<' + dep + ',`' + dep + '`>>', file=fp)
 
         write('*Contact*::', file=fp)
-        write('  - ' + contact, file=fp)
+        for contact in contacts:
```

Comment:
It's a good longterm goal, but many of these contacts are internal gitlab handles, not github, so it would make things worse to treat them all as valid links. And a bunch of Khronos participants don't have github handles, or at least not work-associated ones. So deferred for now.

Reply:
Ah, that makes sense.
Though the XML schema says they have to be reachable, and mentions GitHub specifically...